docs/index.md

```diff
@@ -2,17 +2,37 @@
 layout: default
 ---
 
-# Welcome to the GeoMoose Documentation
```

Comment:
This doesn't render well.  Looks kind of like if org-mode and markdown had a kid.

Reply:
The "layout:default" was in the original doc.  I thought it was a artifact for the doc build system (which I eventually need to look into).  Should I take it out?

Reply:
I mean the whole document renders strangely.  I don't know why or what layout:default does.  Click the "Display the rich diff" button for the file in the "Files Changed" tab of the PR.

It doesn't look like it has any clear structure.  

The ### nodes are all on one line.

Reply:
Hopefully the latest commit to this PR resolves the link/header format issue.